src/main/scala/com/lightbend/kafka/scala/streams/KGroupedTableS.scala

```diff
@@ -26,34 +26,34 @@ class KGroupedTableS[K, V](inner: KGroupedTable[K, V]) {
     inner.count(materialized)
 
   def reduce(adder: (V, V) => V,
-    subtractor: (V, V) => V): KTableS[K, V] = {
+             subTractor: (V, V) => V): KTableS[K, V] = {
```

Comment:
Can we add a word exception here?  The original spelling was correct.

Reply:
Actually, subtractor word is not following a naming convention that's why I changed. so now you want me to add word `exception` instead of `subtractor`?

Reply:
I mean the spelling of `subtractor` is correct, `subTractor` is not.  scalastyle should ignore this line or have a word exception added if that's possible.  I quickly looked at the scalastyle docs and didn't see a way to add words.

Reply:
I have reverted incorrect name.

Reply:
Thanks.